install_config/configuring_authentication.adoc

```diff
@@ -1306,9 +1626,63 @@ described in the OpenID spec. Must use `https`.
 described in the OpenID spec. Must use `https`.
 ====
 
+. Configure which link:http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims[claims] to use: 
++
+----
+claims:
```

Comment:
this snippet is disconnected from the larger example, which makes it difficult to understand.